mkosi/resources/mkosi.md

```diff
@@ -2076,6 +2078,18 @@ current working directory. The following scripts are supported:
   artifacts from `SplitArtifacts=yes` or RPMs built in a build script).
   Note that this script does not use the tools tree even if one is configured.
 
+* If **`mkosi.version`** exists and is executable, it is run during
+  configuration parsing and populates `ImageVersion=` with the output on stdout.
+  This can be used for external version tracking, e.g. with `git describe` or
+  `date '+%Y-%m-%d'` Note that this script is executed on the host system
+  without any sandboxing.
+
+* If **`mkosi.rootpw`** exists and is executable, it is run during
+  configuration parsing and populated `RootPassword=` with the output
+  on stdout. This can be used to randomily generate a password and can
```

Comment:
```suggestion
  on stdout. This can be used to randomly generate a password and can
```
Though this is now possible I'm not sure we should advertise throwing around cleartext password.